packages/web-embed-api/src/index.test.ts

```diff
@@ -138,7 +138,7 @@ describe('FormsortWebEmbed', () => {
     );
```

Comment:
Do you mind extending the tests slightly, so that:

1. They check for the presence of answers passed to the callbacks
2. They cover the newly-added event types, `StepCompleted` and `StepLoaded`?

Both behaviors are pretty critical to anyone using this functionality in the embed, so I'd like to ensure that it's unlikely we break them

Reply:
Add tests